translations/ja.json

```
@@ -597,50 +597,50 @@
+	"The KVM network name. (kvm2 driver only)": "KVM ネットワーク名（kvm2 ドライバーのみ）",
+	"The VM driver crashed. Run 'minikube start --alsologtostderr -v=8' to see the VM driver error message": "VM ドライバーがクラッシュしました。'minikube start --alsologtostderr -v=8' を実行して、VM ドライバーのエラーメッセージを参照してください",
+	"The VM driver exited with an error, and may be corrupt. Run 'minikube start' with --alsologtostderr -v=8 to see the error": "VM ドライバーがエラー停止しました（破損している可能性があります）。'minikube start --alsologtostderr -v=8' を実行して、エラーを参照してください",
+	"The VM that minikube is configured for no longer exists. Run 'minikube delete'": "minikube が設定された VM はもう存在しません。'minikube delete' を実行してください",
+	"The \\\"{{.name}}\\\" container runtime requires CNI": "\\\"{{.name}}\\\" コンテナランタイムは CNI が必要です",
+	"The ambassador addon has stopped working as of v1.23.0, for more details visit: https://github.com/datawire/ambassador-operator/issues/73": "v1.23.0 で ambassador アドオンは機能を停止しました。 詳細はこちらを参照してください: https://github.com/datawire/ambassador-operator/issues/73",
+	"The apiserver listening port": "API サーバーリスニングポート",
 	"The apiserver name which is used in the generated certificate for kubernetes. This can be used if you want to make the apiserver available from outside the machine": "Kubernetes 用に生成された証明書で使用される API サーバー名。マシンの外部から API サーバーを利用できるようにする場合に使用します",
-	"The argument to pass the minikube mount command on start": "起動時に minikube マウント コマンドを渡す引数",
-	"The argument to pass the minikube mount command on start.": "",
-	"The authoritative apiserver hostname for apiserver certificates and connectivity. This can be used if you want to make the apiserver available from outside the machine": "",
-	"The base image to use for docker/podman drivers. Intended for local development.": "",
-	"The certificate hostname provided appears to be invalid (may be a minikube bug, try 'minikube delete')": "",
-	"The cluster dns domain name used in the Kubernetes cluster": "",
-	"The cluster dns domain name used in the kubernetes cluster": "Kubernetes クラスタで使用されるクラスタ DNS ドメイン名",
-	"The cluster {{.cluster}} already exists which means the --nodes parameter will be ignored. Use \"minikube node add\" to add nodes to an existing cluster.": "",
-	"The container runtime to be used (docker, crio, containerd)": "使用されるコンテナ ランタイム（docker、crio、containerd）",
-	"The control plane for \"{{.name}}\" is paused!": "",
-	"The control plane node \"{{.name}}\" does not exist.": "",
-	"The control plane node is not running (state={{.state}})": "",
-	"The control plane node must be running for this command": "",
+	"The argument to pass the minikube mount command on start": "起動時に minikube マウントコマンドを渡す引数",
+	"The argument to pass the minikube mount command on start.": "起動時に minikube マウントコマンドを渡す引数。",
+	"The authoritative apiserver hostname for apiserver certificates and connectivity. This can be used if you want to make the apiserver available from outside the machine": "API サーバーの証明書と接続の為の、権威 API サーバーホスト名。マシン外部から API サーバーに接続できるようにしたい場合に使用します。",
```

```suggestion
	"The authoritative apiserver hostname for apiserver certificates and connectivity. This can be used if you want to make the apiserver available from outside the machine": "API サーバーの証明書と接続のための、権威 API サーバーホスト名。マシン外部から API サーバーに接続できるようにしたい場合に使用します。",
```
This is due to the following policy.
- 「〜なし」「〜のため」「〜すること」もひらがなで記載します。「〜無し」「〜の為」「〜する事」などは避けます。

---

done.